### PR TITLE
feat: Reading supported_servers.json locally if timeout while fetch r…

### PR DESCRIPTION
**Pull request by markiing**
- Reading a local file as fallback when fetch throws exception
- Rolling back `ClientDTO` to use `name` instead of `processName` 
- Renaming `Hexed.cs` to `Client.cs` (For better understanding)

### CODE REVIEW

File: Forms/ClientUpdaterForm.cs [hunk 1]
```diff
@@ -24,22 +25,53 @@ private async void StartUpdate()
         {
             try
             {
+                //If fetch successfully update and load local file.
+                client.Timeout = TimeSpan.FromSeconds(5);
                 string result = await client.GetStringAsync(AppConfig._4RClientsURL);
-                List<ClientDTO> clients = JsonConvert.DeserializeObject<List<ClientDTO>>(result);
-                foreach(ClientDTO clientDTO in clients)
+                LoadServers(result);
+            }catch(Exception)
```
Comment (afonsocdaniel) [inline Forms/ClientUpdaterForm.cs:32]:
```suggestion
            } catch(Exception)
```

**Review by afonsocdaniel — COMMENTED** [review]
Should we add by default the `supported_servers.json` files?

File: Forms/ClientUpdaterForm.cs (hunk 1 above)
Comment (markiing) [reply Forms/ClientUpdaterForm.cs:32]: Maybe we can add as resource file and carry an `fallback-fallback` version. That's right?

Comment (afonsocdaniel) [reply Forms/ClientUpdaterForm.cs:32]: Just an idea so we can avoid users asking over Discord about how to download this file.

Comment (markiing) [reply Forms/ClientUpdaterForm.cs:32]: Fair enought. I'll do that. 

Thanks for the suggestion.

Comment (markiing) [reply Forms/ClientUpdaterForm.cs:32]: done @afonsocdaniel .

**Review by afonsocdaniel — APPROVED** [review]

**Pull request merged**